src/gui/editors/PianoRoll.cpp

```diff
@@ -2837,10 +2845,12 @@ void PianoRoll::paintEvent(QPaintEvent * pe )
 			{
 			case PR_WHITE_KEY_SMALL:
 			case PR_WHITE_KEY_BIG:
-				p.setBrush(pressed ? m_whiteKeyActiveBackground : m_whiteKeyInactiveBackground);
+				if (mapped) {p.setBrush(pressed ? m_whiteKeyActiveBackground : m_whiteKeyInactiveBackground);}
+				else {p.setBrush(m_whiteKeyDisabledBackground);}
```

Comment:
Just to fix whitespace, but the variable names are so long we may want to split the line up. It really explodes it following our conventions, so idk really.
```suggestion
				if (mapped)
				{
					p.setBrush(
						pressed
						? m_whiteKeyActiveBackground
						: m_whiteKeyInactiveBackground
					);
				}
				else { p.setBrush(m_whiteKeyDisabledBackground); }
```

Reply:
What is the problem with whitespace? I don't quite understand what you mean.
According to the conventions, the format with `if` + {} and one statement on the same line is all right. Also the line is just 110 characters long, which fits the limit of 120 characters. And as you show, splitting it up takes way more space, making it harder to read and understand at a glance..

I don't want to be dismissive, but I just don't see the problem. :)

Reply:
Spaces after/before `{}`. And then per coding conventions the very long line should move `{}` to their own lines (though I think we're discussing having the open bracket on the same line) and have `p.setBrush` on its own line. Then with how long the tertiary operator is there it might need to be split up.

Reply:
> Spaces after/before {}.

Oh. I always thought it's a bug in the coding conventions remaining from the "old times", because all the other places enforce the (more sensible) "no spaces between ()".

I think we just have a different preference for line length and therefore interpret the rule allowing "if oneliners" differently. :D I just take it as "if it fits, it is allowed". But maybe it would be better to try and keep most lines under 80 characters and use the 120 char limit only when it is really needed. I don't know. It's an endless discussion..

Reply:
Yeah it's a give and take. Readability is the most important though! I try not to be a style nag, but long lines are hard to determine if they are actually within limits or not.

I think we'll need to define what our preferred tab length is, and then go with that.

Reply:
> I think we'll need to define what our preferred tab length is, and then go with that.

I was of the idea that the tab size used by LMMS is 4 characters and keeping them as tabs (not replaced by spaces). That's why I also aligned using tabs: because there was a defined standard tab size for LMMS.

Reply:
I don't think we ever defined the tab size. 4 is what most of us use though I think. Alignment should be tabs to block, then spaces (or single tab similar to how I suggested).

Reply:
Oh, you are right -- it seems #4810 never actually made it into the conventions. Oh well.. ^^;